examples/4.mint_usdc.ts

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
Users wont be able to mint usdc on our contracts anyways, so example should be removed